FRAMEWORKS.yml

```diff
@@ -99,9 +99,9 @@ crystal:
     website: spider-gazelle.net
     version: "1.1"
 nim:
-  # jester:
-  #   github: dom96/jester
-  #   version: "0.4"
+  jester:
+    github: dom96/jester
+    version: "0.4"
```

Comment:
*0.4.1

Reply:
sure, but in fact this is use on `README` (to display version of each framework).
To clarify/simplify maintainance, I only display them on X.Y format